sbor/src/print/state.rs

```diff
@@ -57,11 +60,11 @@ pub const TITLE_SIZE: usize = 32;
 pub struct ParameterPrinterState<T: Copy> {
     pub display: StaticVec<u8, { DISPLAY_SIZE }>,
     pub data: StaticVec<u8, { PARAMETER_AREA_SIZE }>,
-    pub title: StaticVec<u8, { TITLE_SIZE }>,
+    pub title: StaticVec<u8, { TITLE_SIZE }>, // Intermediate buffer for formatting instruction titles (instruction number)
```

Comment:
docs must go op top of field, or are you "waiting" with that until last minute? just dont forget :)